terraform/cloudfront.tf

```diff
@@ -30,34 +30,27 @@ resource "aws_cloudfront_origin_access_identity" "app" {
 
 
 resource "aws_cloudfront_function" "response" {
+  provider = aws.us-east-1
   name    = "${local.namespace}-cf-response"
   runtime = "cloudfront-js-1.0"
   comment = "Add security headers"
   code    = file("${path.module}/cloudfront/response.js")
 }
 
 resource "aws_cloudfront_function" "request" {
+  provider = aws.us-east-1
   name    = "${local.namespace}-cf-request"
   runtime = "cloudfront-js-1.0"
   comment = "Next request handler"
   code    = file("${path.module}/cloudfront/request.js")
 }
 
 
-resource "aws_s3_bucket" "app_logs" {
-  bucket = "${local.app_name}-logs"
-  acl    = "log-delivery-write"
-}
-
 resource "aws_cloudfront_distribution" "app" {
   comment = local.app_name
 
   aliases = local.has_domain ? [var.domain] : []
 
-  logging_config {
```

Comment:
do we need to remove this?

Reply:
We will try to enable this while working on logging ticket.